actiontest/readme.md

```diff
@@ -0,0 +1,12 @@
+To run locally. 
```

Comment:
I wonder if we could remove this directory and have its contents in the root, and part of the reason for that is the root README.md has a Development heading, where these steps might be more easily discoverable. If we do this we'll also need to update the .gitignore with the new secrets path.

Reply:
👍